spec/integration/rules_spec.rb

```diff
@@ -98,8 +98,9 @@
           )
           user = User.from_x_rh_identity(x_rh_identity[:identity])
           user.save
-          profiles(:one).update(account: user.account, hosts: [hosts(:one)])
-          rules(:one).update(profiles: [profiles(:one)])
+          profiles(:one).update!(account: user.account, hosts: [hosts(:one)])
+          rules(:one).rule_results.destroy_all
+          rules(:one).update!(profiles: [profiles(:one)])
```

Comment:
This `update` was silently failing all the sudden due to the newly associated rule_result fixtures. I've updated them to use `update!` so this kind of thing is easier to spot in the future. The update was failing because the associated rule results were invalid - deleting them from the rule fixed the test.